packages/vats/src/proposals/upgrade-asset-reserve-proposal.js

```diff
@@ -89,5 +89,8 @@ export const getManifestForUpgradingAssetReserve = (
       },
     },
   },
+  installations: {
+    assetReserve: restoreRef(assetReserveRef),
```

Comment:
Again, I suggest `reserve`, not `assetReserve`.

```suggestion
    reserve: restoreRef(assetReserveRef),
```

We can see this in [published.agoricNames.installation](https://vstorage.agoric.net/?path=published.agoricNames.installation&endpoint=https%3A%2F%2Fmain-a.rpc.agoric.net%3A443&height=null) on mainnet (or any of our running networks, I'm pretty sure).

Reply:
done

Reply:
I decided to import `agoricNamesReserved` from `@agoric/vats/src/core/utils.js` to verify that I'm using actual installation names.